tidb-cloud/secure-connections-to-serverless-tier-clusters.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Secure Connections to Serverless Tier Clusters
+summary: Introduce TLS connection in TiDB Serverless Tier clusters.
+---
+
+# Secure Connections to Serverless Tier Clusters
+
+This document introduces the core information about TLS in TiDB Serverless Tier.
+
+## Can I disable TLS in TiDB Serverless Tier?
+
+No.
+
+TiDB Serverless Tier allows only TLS connections and reject non-TLS connections. The reason for this is that users connect to TiDB Serverless Tier clusters through a public network, so communication security is really important.
+
+## What TLS versions can I use?
+
+TiDB Serverless Tier only supports TLS 1.2 and TLS 1.3. 
+
+## What certificates do I need？
+
+TiDB Serverless Tier use certificates from [Let's Encrypt](https://letsencrypt.org/) as a Certificate Authority (CA) for TLS connection between clients and TiDB Serverless Tier Clusters. Usually, Let's Encrypt's root certificate ([ISRG Root X1](https://letsencrypt.org/certs/isrgrootx1.pem.txt)) is present in your system's root CA stores. If the client uses the system's root CA stores by default, such as Java and Go, you can easily connect securely to TiDB Serverless Tier clusters without specifying the path of CA roots.
+
+However, some drivers and ORMs do not use the system root CA stores. In those cases, you should configure CA root path of the drivers or ORMs to your system root CA stores. For example, when you use MySQLdb to connect TiDB Serverless Tier clusters in Python, you should set `ca: /etc/ssl/cert.pem` in ssl configurations.
+
+> **Note:**
+> 
+> TiDB Serverless Tier does not provide a CA root certificate download, because we don't guarantee that the same CA will be used to issue a certificate in the future, which will cause the CA root certificate to change. 
+> While, TiDB Serverless Tier promise to always use the CA root certificate that is normally available, which is provided in all common systems. 
+> 
+> If you really need the CA certificate of TiDB Serverless Clusters, we recommend that you download the [Mozilla CA Certificate bundle](https://curl.se/docs/caextract.html) instead of a single CA certificate.
```

Comment:
```suggestion
> If you really need the CA certificate of a TiDB Serverless Tier cluster, it is recommended that you download the [Mozilla CA Certificate bundle](https://curl.se/docs/caextract.html) instead of a single CA certificate.
```